### PR TITLE
Update new merge algorithm information

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/refguide/modeling/menus/view-menu/changes-pane.md [hunk 1]
```diff
@@ -98,7 +98,9 @@ The toolbar at this level contains the following buttons:
 
 * **Back** – takes you back to the top level
 * **Go to** – takes you directly to the selected element
-* **Merge** – start the merge process — instructions for resolving conflicts are in the document [New Merge Algorithm with Fine-Grained Conflict Resolution](/refguide/new-merge-algorithm/) 
+* **Interactive Merge** – combines changes where possible for all conflicting documents that you can revise in the [Merge Mode](#merge-mode); for more information on how to solve conflicts, see [Merge Algorithm with Fine-Grained Conflict Resolution](/refguide/merge-algorithm/)
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/view-menu/changes-pane.md:101]: Is description of **Interactive Merge**, **Resolve conflict using my whole document**, **Resolve conflict using their whole document** correct?

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md [hunk 2]
```diff
@@ -53,33 +38,33 @@ You make the following changes on a branch line:
...
 2. A **list order conflict**. Both of you added widgets to the bottom layout grid. The merge algorithm cannot guess the right order for the two new widgets and it reports the list order conflict. This is a reminder for you (the developer who is doing the merge) to look at the final layout and confirm the order. 
 
-    {{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/new-merge-algorithm/new-merge-algorithm-conflicts.png" alt="New algorithm conflicts" >}}
+    {{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm/merge-algorithm-conflicts.png" alt="Merge algorithm conflicts" >}}
 
-To start the resolution process, click the **Merge** button. The page is opened in a special mode with an orange bar at the top:
+To start solving conflicts first choose an approach for all conflicts that merge algorithm will apply:
 
-{{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/new-merge-algorithm/new-merge-algorithm-orange-tab.png" alt="Document with orange bar" >}}
+* **Interactive Merge** – combines both changes where possible
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md:55]: Is description of **Interactive Merge**, **Resolve conflict using my whole document**, **Resolve conflict using their whole document** correct?

Comment (tknippenberg) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md:55]: I think it is correct. @mikita-haurykau, what do you think?

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md:55]: Agreed with Tom via Slack that I can merge it.

**Pull request merged**